client/blocks/product-purchase-features-list/mobile-apps.jsx

```diff
@@ -20,7 +20,7 @@ export default localize( ( { translate } ) => {
 				icon={ <img alt="" src="/calypso/images/illustrations/jetpack-apps.svg" /> }
 				title={ translate( 'Download our mobile apps' ) }
 				description={ translate(
-					'Manage all your sites from a single dashboard: publish content, ' +
+					'Manage all of your sites from a single dashboard: publish content, ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Manage all your sites from a single dashboard: publish content, track stats, moderate comments, and more.' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).